politeiawww/middleware.go

```diff
@@ -137,3 +139,57 @@ func recoverMiddleware(next http.Handler) http.Handler {
 		next.ServeHTTP(w, r)
 	})
 }
+
```

Comment:
These functions aren't really middleware and they are specific to cms.  They should go in the cmswww.go file.